src/qibolab/emulator.py

```diff
@@ -0,0 +1,56 @@
+import pathlib
+
+from qibolab.channels import ChannelMap
+from qibolab.instruments.simulator.pulse_simulator import PulseSimulator
+from qibolab.platform import Platform
+from qibolab.serialize import load_qubits, load_runcard, load_settings
+
+
+def create_runcard_emulator(runcard_folder: str, pulse_simulator: PulseSimulator):
+    """Create a emulator platform using the emulator instrument."""
+    # load runcard
+    runcard_folder_path = pathlib.Path(runcard_folder)
+    original_runcard = load_runcard(runcard_folder_path)
+    runcard = load_runcard(runcard_folder_path)
+
+    # todo: check PulseSimulator topology is superset of runcard topology
+
+    model_config = pulse_simulator.model_config
+    emulator_name = pulse_simulator.emulator_name
+    qubits_list = model_config["qubits_list"]
+    couplers_list = model_config["couplers_list"]
+    runcard_qubits_list = original_runcard["qubits"]
+    runcard_couplers_list = original_runcard["couplers"]
+
+    print(emulator_name)
+    print("emulator qubits: ", qubits_list)
+    print("emulator couplers: ", couplers_list)
+    print("runcard qubits: ", runcard_qubits_list)
+    print("runcard couplers: ", runcard_couplers_list)
+    print(f"sampling rate: {pulse_simulator.sampling_rate}GHz")
+    print(f"simulation sampling boost: {pulse_simulator.sim_sampling_boost}")
```

Comment:
These are probably not needed, or if you like to keep them maybe we should convert to `log` instead of `print`.

Reply:
Please, keep the conversation open until solved, and (if possible) add in the last comment the hash of the commit solving it.

Reply:
> I will switch to log for now.

Fixed in e035cb3